app/settings.py

```diff
@@ -68,3 +70,10 @@
 
 STATICFILES_DIRS = ()
 STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+
+REST_FRAMEWORK = {
+    'UNICODE_JSON': False,
```

Comment:
This is a bit annoying. Basically, even though our API standards say we should [use UTF-8](https://github.com/18F/api-standards#use-utf-8) and set our `Content-Type` header to `Content-Type: application/json; charset=utf-8`, what's really weird is that [doing so actually goes against the JSON spec](http://lucumr.pocoo.org/2013/7/19/application-mimetypes-and-encodings/).

There's a lot of contention over whether or not it's ok to specify a `charset` parameter for JSON content, and personally I think the easiest solution is to avoid the debate altogether by ensuring that your output is ASCII with unicode escape sequences in them. This does result in larger payloads and is especially inefficient if our payloads contain lots of non-ASCII characters, but it's the most compatible across clients.

Anyways, the `UNICODE_JSON` parameter above makes that happen.